includes/data-events/README.md

```diff
@@ -130,6 +130,24 @@ When there's a new donation, either through Stripe or Newspack (WooCommerce) pla
 | `subscription_id` | `int`    | The related subscription id (if any)                   |
 | `platform_data`   | `array`  |                                                        |
 
+### `order_completed`
+
+When any WooCommerce order is completed with a successful payment.
+
+| Name              | Type     | Obs                                                    |
+| ----------------- | -------- | ------------------------------------------------------ |
+| `user_id`         | `int`    |                                                        |
+| `email`           | `string` |                                                        |
+| `amount`          | `float`  |                                                        |
+| `currency`        | `string` |                                                        |
+| `recurrence`      | `string` |                                                        |
```

Comment:
Given that the `recurrence` data is only available for donation orders, I suggest we don't add it to this action to avoid confusion. We can leave `donation_new` and the other existing donation-related actions to provide this data.

Reply:
Sounds good—removed in 74b8fb80